test/src/userobjects/RestartableTypes.C

```diff
@@ -17,7 +17,7 @@
 template<>
 InputParameters validParams<RestartableTypes>()
 {
-  InputParameters params = validParams<UserObject>();
+  InputParameters params = validParams<GeneralUserObject>();
```

Comment:
These fixes don't appear to be related to this PR. Is this left over stuff from your Warehouse refactor?